hca/constants.py

```diff
@@ -0,0 +1,9 @@
+"""File to keep constants together."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+
+class Constants:
+    """Class to keep constants together."""
+
+    OBJECT_SPLITTER = "/"
```

Comment:
Will add. It's the character we decided on to split the variables within an object when they're listed. So in the put-bundles command, the files should be specified as `<INDEXED><OBJECT_SPLITTER><NAME><OBJECT_SPLITTER><UUID><OBJECT_SPLITTER><VERSION>`